_posts/2023-09-13-prospero-add-feature-pack.adoc

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title:  "Installing additional feature packs in WIldfly"
+date:   2023-09-13
+tags:   prospero galleon wildfly
+author: spyrkob
+description: Using Prospero to installing additional feature packs in existing Wildfly servers.
+---
+
+== Additional feature packs for Wildfly
+
+Wildfly server capabilities can be extended by installing additional feature packs such as wildfly-datasources-galleon-pack or wildfly-myfaces-galleon-pack.
```

Comment:
could you link to https://github.com/wildfly-extras/wildfly-datasources-galleon-pack & https://github.com/wildfly-extras/wildfly-myfaces-feature-pack?